packages/firestore/src/local/local_store.ts

```diff
@@ -439,7 +430,6 @@ export class LocalStore {
             // Do not ref/unref unassigned targetIds - it may lead to leaks.
             let queryData = this.queryDataByTarget[targetId];
             if (!queryData) return;
-
```

Comment:
I would prefer if we keep this newline. It makes the early return stand out more.